XcodeServerSDKTests/Casettes/get_devices.json

```diff
@@ -0,0 +1,366 @@
+              "_rev" : "3-834fbad0ab85ac3ba3211f237e7d7c27",
+              "architecture" : "x86_64",
+              "isServer" : false,
+              "retina" : false,
+              "tinyID" : "1563795"
+            },
+            {
+              "osVersion" : "2.0",
+              "connected" : true,
+              "simulator" : true,
+              "modelCode" : "Watch1,1",
+              "deviceType" : "com.apple.iphone-simulator",
+              "modelName" : "Apple Watch - 38mm",
+              "_id" : "1ad0e8785cacca73d980cdb236009347",
+              "modelUTI" : "com.apple.watch-38mm",
+              "doc_type" : "device",
+              "trusted" : true,
+              "name" : "Apple Watch - 38mm",
+              "supported" : true,
+              "identifier" : "7358ED86-5653-46D1-BBFF-DB0C40C47CC2",
+              "enabledForDevelopment" : true,
+              "platformIdentifier" : "com.apple.platform.watchsimulator",
+              "_rev" : "3-a1e925ad4b3b5c769b8e5bfb8ae9e313",
+              "architecture" : "i386",
+              "isServer" : false,
+              "retina" : false,
+              "tinyID" : "BD56E7F"
+            },
+            {
+              "activeProxiedDevice" : {
+                "retina" : false,
+                "connected" : true,
+                "modelCode" : "Watch1,1",
+                "simulator" : true,
+                "osVersion" : "2.0",
+                "modelName" : "Apple Watch - 38mm",
+                "deviceType" : "com.apple.iphone-simulator",
+                "supported" : true,
+                "identifier" : "7358ED86-5653-46D1-BBFF-DB0C40C47CC2",
+                "enabledForDevelopment" : true,
+                "architecture" : "i386",
+                "isServer" : false,
+                "doc_type" : "device",
+                "trusted" : true,
+                "platformIdentifier" : "com.apple.platform.watchsimulator",
+                "name" : "Apple Watch - 38mm",
+                "modelUTI" : "com.apple.watch-38mm"
+              },
+              "osVersion" : "9.0",
+              "connected" : true,
+              "simulator" : true,
+              "modelCode" : "iPhone7,2",
+              "deviceType" : "com.apple.iphone-simulator",
+              "modelName" : "iPhone 6",
+              "_id" : "1ad0e8785cacca73d980cdb236009a54",
+              "modelUTI" : "com.apple.iphone-6-b4b5b9",
+              "doc_type" : "device",
+              "trusted" : true,
+              "name" : "iPhone 6",
+              "supported" : true,
+              "identifier" : "29DEF144-732D-4D3E-862E-82120F688B64",
+              "enabledForDevelopment" : true,
+              "platformIdentifier" : "com.apple.platform.iphonesimulator",
+              "_rev" : "4-927f35acc0dd9519dc078529f32f1079",
+              "architecture" : "x86_64",
+              "isServer" : false,
+              "retina" : false,
+              "tinyID" : "76AFAC6"
+            }
```

Comment:
I think it's not a good sign to share with public identifiers and so on... 😉 I guess we've no other possibility to achieve this test with simulators.